src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -795,7 +795,7 @@ struct SharedGlobals {
 
 static SharedGlobals GVars;
 
-markWord ObjectSynchronizer::read_stable_mark(oop obj) {
+static markWord read_stable_mark(oop obj) {
```

Comment:
Do you still need this method, or? If you still need it, let's not put it outside the `ObjectSynchronizer` "namespace".

Reply:
Originally, read_stable_mark() is static in upstream. A while back I put it into the ObjectSynchronizer namespace to reuse it in Shenandoah, but that is no longer needed, so I reverted the scope back to compilation-unit-private. Yes, we will still need it, when we make Lilliput switchable, but I'd keep it compilation-unit-private for the reasons mentioned.

Reply:
All right then!